src/common/util.ts

```diff
@@ -40,21 +39,28 @@ export const trimSlashes = (url: string): string => {
   return url.replace(/^\/+|\/+$/g, "")
 }
 
+/**
+ * Resolve a relative base against the window location. This is used for
+ * anything that doesn't work with a relative path.
+ */
+export const resolveBase = (base?: string): string => {
+  // After resolving the base will either start with / or be an empty string.
+  if (!base || base.startsWith("/")) {
+    return base || ""
+  }
+  const parts = location.pathname.replace(/^\//g, "").split("/")
```

Comment:
Looks to me like we don't need `.replace(/^\//g, "")` here because we return if `base.startsWith("/")`

Reply:
Oh snap good catch.